src/components/Folder.js

```diff
@@ -0,0 +1,109 @@
+import Header from './Header';
+import Footer from './Footer';
+import '../css/Reset.css';
+import '../css/Global.css';
+import '../css/Folder.css';
+import React, { useState, useEffect } from 'react';
+import ShareAPI from './share/ShareAPI';
+import CountDate from './share/CountDate';
+import { Link } from 'react-router-dom';
+
+function Folder(){
+  const [folderData, setFolderData] = useState(null);
+
+  useEffect(() => {
+    async function fetchData() {
+      const data = await ShareAPI('https://bootcamp-api.codeit.kr/api/sample/folder');
+      setFolderData(data); // 데이터를 상태에 저장
+    }
+    
+    fetchData();
+  }, []); // 빈 배열을 넘겨주어 컴포넌트가 마운트될 때 한 번만 실행
+
+  const CardContents = () =>{ // 카드 목록에 값이 1개씩 들어가게하기 위해 만든 함수
```

Comment:
P3:
주석은 관련 scope 최상단에 적어주시면 좋을 것 같습니다.

```suggestion
 // 카드 목록에 값이 1개씩 들어가게하기 위해 만든 함수
  const CardContents = () =>{
```

Reply:
P2:
CardContents는 따로 컴포넌트로 빼면 더 가독성에 좋을 것 같아요.